### PR TITLE
feat: Add Session Sections

### LINKED ISSUE #1646: Add Session Section
**Describe the feature you'd like**
Show a list of all the sessions organized by the user along with track speaker and other information regarding the track

**Would you like to work on the issue?**
- [x] Yes

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
Fixes #1646 

Changes: Added Session Sections along with micro-location and session-type

Screenshots for the change:
screenshot-1556030469147

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
@iamareebjamal need some suggestion for the layout horizontal recyclerview or vertical recyclerview for the list of session

**Comment by iamareebjamal** [comment]
Good job but this is a major addition. It will take some time for me to get back on it

**Comment by theLimitBreaker** [comment]
Shouldn't be there different fragment for sessions info . All info in event Details frag is too much info.

Just have session option -> clicked -> session fragment. More concentrated on sessions.

I agree it's major work .

**Comment by aggarwalpulkit596** [comment]
@theLimitBreaker I'm thinking of redesigning the entire event detail fragment after this PR is merged with Tab Layout and also there will be seperate detailed fragment for the session along with track details and speaker associated with the session and any other data in relationship with session

**Comment by theLimitBreaker** [comment]
Tab layout is great. But if you first merge this PR. It would make the EventDetailsFragment excessive  information in fragment.
Also if you make fragment right away now. You could use it in tab layout when you redesign
**Benefit**
- Even after this PR merged UX would be good, and no excessive information
- When you redesign , use these fragments again

**Comment by angmas1** [comment]
> @theLimitBreaker I'm thinking of redesigning the entire event detail fragment after this PR is merged with Tab Layout and also there will be seperate detailed fragment for the session along with track details and speaker associated with the session and any other data in relationship with session

Can I work on adding separate sessions fragment if you don't mind?

**Comment by aggarwalpulkit596** [comment]
@angmas1 I've already done that work just waiting for this PR to be merged first

**Comment by aggarwalpulkit596** [comment]
@iamareebjamal any update for this ?

File: app/src/main/res/layout/item_session.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            android:ellipsize="end"
+            android:lines="1"
+            tools:text="Experience"
+            android:textSize="@dimen/text_size_large" />
+
+        <TextView
+            android:id="@+id/mircolocation"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="16dp"
```
Comment (nikit19) [inline app/src/main/res/layout/item_session.xml]: in dimens.xml

**Comment by aggarwalpulkit596** [comment]
@iamareebjamal @nikit19 resolved conflicts

**Comment by aggarwalpulkit596** [comment]
@nikit19 anything left?

**Comment by nikit19** [comment]
@aggarwalpulkit596 looks good to me. 
Let @iamareebjamal review, will merge after that

**Comment by iamareebjamal** [comment]
Hide the calendar icons if there is no info

**Comment by aggarwalpulkit596** [comment]
@iamareebjamal actually I wanted to ask that how to get the time according to current time zone

**Comment by aggarwalpulkit596** [comment]
@iamareebjamal updated

File: app/build.gradle [hunk 2]
```diff
@@ -36,11 +36,11 @@ android {
             shrinkResources true
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-            buildConfigField "String", "DEFAULT_BASE_URL", '"https://api.eventyay.com/v1/"'
+            buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
             buildConfigField "String", "MAPBOX_KEY", '"'+MAPBOX_KEY+'"'
         }
         debug {
-            buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
+            buildConfigField "String", "DEFAULT_BASE_URL", '"https://api.eventyay.com/v1/"'
```
Comment (iamareebjamal) [inline app/build.gradle]: Revert

Comment (aggarwalpulkit596) [reply app/build.gradle]: @iamareebjamal updated

**Pull request merged**